src/app/helpers/apiWhiteList.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+/**
+ * @file - Handle whitelisting of Portal API(s)
+ * @since release-3.1.0
+ * @version 1.0
+ */
+
+const _           = require('lodash');
+const logger      = require('sb_logger_util_v2');
+const uuidv1      = require('uuid/v1');
+const dateFormat  = require('dateformat');
+const ROUTE       = require('./whitelistApis');
+
+/**
+ * @description - Function to check whether incoming API is whitelisted (or) not
+ * 1. If incoming API is present in API_LISTS (whitelistApis); then request is passed further
+ * 2. If API is does not exists then request is terminated and 401 UNAUTHORIZED response is sent
+ * @since release-3.1.0
+ */
+const isAPIWhiteListed = () => {
```

Comment:
Change the name to isValid or IsExist

Reply:
I would recommend having method name as **isAllowed**; which conveys whether API is allowed or not.
Please confirm on this.
_P.S. Awaiting response._

Reply:
Changes updated.